docs/index.rst

```diff
@@ -36,36 +36,45 @@ Multi-vector simulator
 
-2.	Energy consumption, which is expressed as times series based on the type of energy (in this case: electrical and therma);
+*	Energy consumption, which is expressed as times series based on the type of energy (in this case: electrical and therma);
 
-3.	System configuration, in which the user specifies the technical and financial data of each asset; and
+*	System configuration, in which the user specifies the technical and financial data of each asset; and
 
-4.	Meteorological data, which is related to the components that generate electricity by harnessing an existing source of energy that is weather- and time-dependent (e.g., solar and wind power).
+*	Meteorological data, which is related to the components that generate electricity by harnessing an existing source of energy that is weather- and time-dependent (e.g., solar and wind power).
 
-This set of input data is then translated to a linear programming problem, also known as a constrained optimization problem. The MVS is based on the oemof-solph python library that describes the problem by specifying the objective function, the decision variables and the bounds and constraints. The goal is to (1) minimize the production costs by determining the generating units' optimal output, which meets the total demand, and (2) optimize near-future investments in generation and storage assets with the least possible cost of energy.
-The simulation outputs are also separated into categories: the economic results used for the financial evaluation, such as the levelized cost of electricity or heat or the net present value of the projected investments, the technical results that include the optimized capacities and dispatch of each asset for instance, and the system’s environmental contribution in terms of CO2 emissions. All these results are valuable for the decision making.
+This set of input data is then translated to a linear programming problem, also known as a constrained optimization problem. The MVS is based on the `oemof-solph <https://github.com/oemof/oemof-solph>`__ python library that describes the problem by specifying an objective function to minimize, the decision variables and the bounds and constraints.
 
+The simulation outputs are also separated into categories:
 
-Maintainers
-===========
+* Economic results used for the financial evaluation, such as the levelized cost of electricity/heat or the net present value of the projected investments
+* Technical results that include the optimized capacities and dispatch of each asset or the system’s environmental contribution in terms of CO2 emissions.
 
-Say who develop and maintains this
 
-Say a word about funding
+Maintainers
+===========
 
-Origin
-======
+The multi-vector simulator is currently maintained by staff from `Reiner Lemoine Institute <https://reiner-lemoine-institut.de/>`__.
 
-What started MVS
+It is developed as a work package in the European Union’s Horizon 2020 Research `E-Land project <https://elandh2020.eu/>`__
```

Comment:
```suggestion
The MVS is developed as a work package in the European Union’s Horizon 2020 Research `E-Land project <https://elandh2020.eu/>`__
:Acknowledgement: This project has received funding from the European Union’s Horizon 2020 Research and Innovation programme under Grant Agreement No 824388. 
:Disclaimer: The information and views set out in this poster are those of the author(s) and do not necessarily reflect the official opinion of the European Union. Neither the European Union institutions and bodies nor any person acting on their behalf may be held responsible for the use which may be made of the information contained herein
```
We also need to include the European Flag here: https://europa.eu/european-union/about-eu/symbols/flag_en

Reply:
I hope this location is prominent enough for the acknowledgement.